homeworks/roman.lytvynenko_Lytvynenko18/GamesProject/style.css

```diff
@@ -26,3 +26,62 @@
 .about-me__links {
   color: #EF4934;
 }
+
+.about-me__article {
+  display: grid;
+  grid-template-columns: auto auto;
+  width: 1175px;
+  height: 479px;
+  left: 53px;
+  border-radius: 22px;
+}
+
+.about-me__picture {
+  width: 532px;
+  height: 356px;
+  left: 53px;
+  margin-top: 48px;
+  margin-left: 53px;
+  border-radius: 22px;
+}
+
+.about-me__header {
+  display: grid;
+  grid-template-columns: 10fr 1fr 1fr 1fr;
+  align-content: center;
+  background-color: #EF4934;
+  width: 1280px;
```

Comment:
maybe you will not fix the width of the header?
![image](https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/9d75291d-90fc-4c43-9b6c-b7d19ec8eb53)
![image](https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/995caefc-1004-4d77-be3b-e0594f1308eb)

Reply:
fixed in 5015c401e868e8132d82c925762abaffb9ef6e48